pkg/driver/controller_server.go

```diff
@@ -105,10 +105,10 @@ func (d *Driver) CreateVolume(ctx context.Context, req *csi.CreateVolumeRequest)
 	availableSize := int64(quota.DiskGigabytesLimit - quota.DiskGigabytesUsage)
 	if availableSize < desiredSize {
 		log.Error().Msg("Requested volume would exceed storage quota available")
-		return nil, status.Error(codes.OutOfRange, fmt.Sprintf("Requested volume would exceed volume space quota by %d GB", desiredSize-availableSize))
```

Comment:
> [!NOTE]
I changed it to pass the arguments directory to` status.Errorf`, which achieves the same result.
This makes the implementation simpler.